File: api/code-samples/python/klay/transaction/getTransactionByHash.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

transactionHash = "0xaca5d9a1ed8b86b1ef61431b2bedfc99a66eaefc3a7e1cffdf9ff53653956a67"

sdk = OpenSDK(host)
klay_response = sdk.klay.get_transaction_by_hash(transactionHash)

print(json.loads(klay_response.response.data))

File: api/code-samples/javascript/klay/transaction/getTransactionByHash.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const transactionHash = '0x21b2919b89278ca786226f10edbaadced7381dbd73df546a4823547aaebffa58'

    sdk.klay.getTransactionByHash(transactionHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()